man/runc-ps.8.md

```diff
@@ -1,13 +1,21 @@
+% RUNC-PS(8) runc-ps Man Pages
+
 # NAME
-   runc ps - ps displays the processes running inside a container
+**runc** **ps** - ps displays the processes running inside a container
 
 # SYNOPSIS
-   runc ps [command options] <container-id> [ps options]
+**runc** **ps** [_command options_] <_container-id_> [_format descriptors_]
```

Comment:
`runc ps` no longer accept `[ps options]`?
This would break compatibility.

Reply:
That's correct, with the proposed change `runc ps` would not accept `[ps options]` anymore. This is because the `ps` command execution is replace with calls to the  psgo API, and psgo is not meant to be a clone of `ps`.

Reply:
can we fallback to the `ps` mode when a ps option is specified?

Reply:
It looks like a good compromise. Should we fallbak to `ps` also when no options are specified?

@vrothberg WDYT?

Reply:
I understand the request but I don't think that's very user-friendly and hard to explain (and hard to understand) when `runc-ps` would return which kind of output in which kind of format.

In case of podman we decided to break the compatibility with flags but gained the flexibility of extracting much more data that comes in handy when debugging (e.g., seccomp, security labels, capabilities, host pid/user/group, etc.).

I *think* that it's a good trade-off but I don't know if there are users in the wild relying on how `runc-ps` works now. As there's not yet a v1.0.0, we might be able to change it :)

Reply:
@AkihiroSuda Do you know of anyone using `runc ps -- -ef` or similar options? This is an issue I was aware of when I proposed doing this to @marcov, but given the history of this feature I'm not sure it's widely used.

(One other possible solution would be to take a look at how `ps` does the mapping of flags to fields and we could just do that formatting for users -- since `psgo` supports `ps`-like formatting flags.)

Reply:
cc @Random-Liu

Reply:
Friendly ping.

Reply:
/cc @opencontainers/runc-maintainers 

Do we need to support ps flags? In which case would we be fine with having a dumb conversion from ps flags to the columns we need to ask psgo to give us?

Reply:
I think need to support PS flags, especially when there is no PS command in a container. You can see https://github.com/opencontainers/runc/pull/1879